src/documentation/common.md

```diff
@@ -86,7 +86,7 @@ Train             |  11.9
 Tramway           |   4
 
 The following fallback modes are also added to the model (they're usually not
-referenced by any Vehicle Journey).
+referenced by any trip).
```

Comment:
we definitively choose between vehicle journey or trip
@prhod @papailio ?

Reply:
Not necessarily everywhere but I remember using the same word in the same document in our last (pair programming) PR with @prhod. This one must have been slipped out but the idea was to have the same word all over this document.